### DISCUSSION TITLE
Move the mobx PropTypes into separate package?

### DISCUSSION THREAD

**Discussion by Twipped** [mobxjs/mobx#2814] [Ideas]
We're using mobx-react-lite in our projects now, having entirely adopted functional components. I'd like to be able to use the PropTypes from mobx-react, but they appear to be bundled in wholesale with the rest of the package, which we don't need included in our bundle.

Would it be possible to get the this extracted as a separate package or module so that they can be used without the rest of the code?

**Comment by danielkcz** [comment]
We are already tracking this for a while in #2541. Feel free to work on that :)

**Reply by iChenLei** [reply]
2019's issue 😄

**Comment by mweststrate** [comment]
Just waiting for a champion :)
